server/index.py

```diff
@@ -0,0 +1,333 @@
+import traceback
+import sieve
+import json
+import shutil
+from moviepy.editor import VideoFileClip, AudioFileClip
+
+from daily import fetch_recordings, get_access_link
+from quart_cors import cors
+import quart
+import requests
+from config import ensure_dirs, get_output_dir_path, get_upload_dir_path
+from quart import Quart, jsonify, send_from_directory
+
+app = Quart(__name__)
+cors(app, allow_origin="http://localhost:5173")
+ensure_dirs()
+app.config["MAX_CONTENT_LENGTH"]=300777216
+
+
+@app.route('/text_to_video_lipsync/<recording_id>', methods=['POST'])
+async def process_recording_text_to_video_lipsync(recording_id):
+    """Processes a Daily recording by given recording ID."""
+    access_link = get_access_link(recording_id)
+
+    try:
+        data = requests.get(access_link, timeout=10)
+    except Exception as e:
+        return process_error('failed to download Daily recording', e)
+
+    file_path = get_new_video_path(recording_id, 'mp4')
+    try:
+        with open(file_path, 'wb') as file:
+            file.write(data.content)
+    except Exception as e:
+        return process_error('failed to save Daily recording file', e)
+
+    return process_text_to_video_lipsync(recording_id, file_path)
+
+@app.route('/audio_enhance/<recording_id>', methods=['POST'])
+async def process_recording_audio_enhance(recording_id):
+    """Processes a Daily recording by given recording ID."""
+    access_link = get_access_link(recording_id)
+
+    try:
+        data = requests.get(access_link, timeout=10)
+    except Exception as e:
+        return process_error('failed to download Daily recording', e)
+
+    file_path = get_new_video_path(recording_id, 'mp4')
+    try:
+        with open(file_path, 'wb') as file:
+            file.write(data.content)
+    except Exception as e:
+        return process_error('failed to save Daily recording file', e)
+
+    return process_audio_enhance(recording_id, file_path)
+
+@app.route('/video_dubbing/<recording_id>', methods=['POST'])
+async def process_recording_video_dubbing(recording_id):
+    """Processes a Daily recording by given recording ID."""
+    access_link = get_access_link(recording_id)
+
+    try:
+        data = requests.get(access_link, timeout=10)
+    except Exception as e:
+        return process_error('failed to download Daily recording', e)
+
+    file_path = get_new_video_path(recording_id, 'mp4')
+    try:
+        with open(file_path, 'wb') as file:
+            file.write(data.content)
+    except Exception as e:
+        return process_error('failed to save Daily recording file', e)
+
+    return process_video_dubbing(recording_id, file_path)
+
+def process_audio_enhance(recording_id: str, video_path: str):
+    try:
+
+        video = sieve.Video(path=video_path)
+        audio = extract_audio(video)
+        print(audio)
+        output = audio_enhance(audio)
+        
+        json_dict = {
+            recording_id: recording_id
+        }
+        
+        print(output.path)
```

Comment:
Does not need to block this PR, I am just going to unresolved this for my own reference in case it makes sense to go back and pick up the straggling comments next week :)